macros/sfdc_formula_view.sql

```diff
@@ -14,9 +14,14 @@
 */
 
 {% if full_statement_version %}
+{% if using_quoted_identifiers %}
+{%- set table_results = dbt_utils.get_column_values(table=source(source_name, 'fivetran_formula_model'), column='"model"', where="\"object\" = '" ~ source_table ~ "'") -%}
```

Comment:
so the models in Snowflake are processed as double-quoted in the `fivetran_formula_model` seed file?
not quite sure I understand the where condition--so the object is also being double-quoted, and it's referencing the object fields in the seed file, and those objects have to be source_tables, but they're being double-quoted in Snowflake, so additional double-quotes are needed? what are the `\` for?